src/containers/Tenant/Diagnostics/Overview/Overview.scss

```diff
@@ -1,13 +1,35 @@
-.kv-tenant-overview {
+@import '../../../../styles/mixins.scss';
+
+.ydb-diagnostics-overview {
```

Comment:
I want to reuse these styles for TableIndexInfo further, so I moved them on the level of Overview component